src/TransformUtils.ts

```diff
@@ -5,6 +5,11 @@ import * as TS from 'typescript';
 // inside the ConfigSet right
 export interface ConfigSet {
   compilerModule: typeof TS;
+  tsCompiler: TsCompiler;
```

Comment:
probably later `ts-jest` can improve `ConfigSet` typings to only expose necessary typings and we won't need this one anymore. Right now it was already improved a lot

```
export declare class ConfigSet {
    readonly parentOptions?: TsJestGlobalOptions | undefined;
    get versions(): Record<string, string>;
    get tsCompiler(): TsCompiler;
    get tsJestDigest(): string;
    readonly logger: Logger;
    constructor(jestConfig: Config.ProjectConfig, parentOptions?: TsJestGlobalOptions | undefined, parentLogger?: Logger);
}
```

Reply:
Yeah, that's great! I think I once copied the own version, as it introduced some babel dependencies.

Reply:
maybe using `import type` from TypeScript 3.8 can avoid those ?

Reply:
Yeah, the babel issue is already resolved, I will reference the ConfigSet from `ts-jest` in all transformers :+1: